homeassistant/components/remember_the_milk.py

```diff
@@ -0,0 +1,267 @@
+"""Component to interact with Remember The Milk.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/remember_the_milk/
+
+Minimum viable product, it currently only support creating new tasks in your
+Remember The Milk (https://www.rememberthemilk.com/) account.
+
+Ideas for improvement:
+- add interface to resolve tasks
+- add option to add a task to a certain list
+- add option to list tasks stored in RTM
+- make registration dialog nicer
+"""
+import logging
+import os
+import json
+import voluptuous as vol
+from homeassistant.const import (CONF_API_KEY, STATE_OK)
+import homeassistant.helpers.config_validation as cv
+
+from homeassistant.helpers.entity_component import EntityComponent
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['RtmAPI==0.7.0', 'httplib2==0.10.3']
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'remember_the_milk'
+DEFAULT_NAME = DOMAIN
+GROUP_NAME_RTM = 'remember the milk accounts'
+
+CONF_SHARED_SECRET = 'shared_secret'
+
+RTM_SCHEMA = vol.Schema({
+    vol.Required(CONF_API_KEY): cv.string,
+    vol.Required(CONF_SHARED_SECRET): cv.string,
+})
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: {
+        cv.string: RTM_SCHEMA
```

Comment:
```yaml
remember_the_milk:
  your_rtm_account:
    api_key: <your secret api key goes here>
    shared_secret: <your secret shared secret goes here>
```

use 

```yaml
remember_the_milk:
  account_name: <your account name>
  api_key: <your secret api key goes here>
  shared_secret: <your secret shared secret goes here>
```

The first configuration style indicates that you can have multiple accounts but according to the docs this is not the case.

Reply:
Actually the idea is to allow several accounts in parallel. I updated the code and the documentation.

Reply:
in that case, you still don't have to use a dictionary but can use `cv.ensure_list`. That means that people can specify 1 account or a list of accounts.

That being said, why would someone have more than 1 account 🤔

Reply:
@balloob: My wife and I both have Todoist accounts; it’s not unlikely that a single household could have multiple accounts with the same service.

Reply:
Good point :)

Reply:
OK